src/domain/node.ts

```diff
@@ -1,19 +1,7 @@
-import { Commands } from "@bc/domain/commands";
-import { ArchiveArtifacts, Mapping } from "@kie/build-chain-configuration-reader";
+import { Node } from "@kie/build-chain-configuration-reader";
```

Comment:
is this file really needed? :thinking:

Reply:
just for the default node value? should i remove it?